packages/theme/index.js

```diff
@@ -1,13 +1,70 @@
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+
+const capitalize = name => name.charAt(0).toUpperCase() + name.slice(1);
+const capEachWord = text => text.split(/[-_\s]/).map(capitalize).join('');
+const ilibVar = name => `ILIB_${name.toUpperCase().replace(/[-_\s]/g, '_')}_PATH`;
+
+const sourceFiles = (dir, action) => {
+	const list = fs.readdirSync(dir);
+	list.forEach(item => {
+		const itemPath = path.join(dir, item);
+		if (fs.statSync(itemPath).isDirectory()) {
+			sourceFiles(itemPath, action);
+		} else if (/\.(css|less|js|jsx|md)$/.test(item)) {
```

Comment:
Should we update this to include `ts|tsx` for future-proofing?

Reply:
Good idea, will do